ui/oopsyraidsy/data/06-ew/trial/rubicante.ts

```diff
@@ -0,0 +1,39 @@
+import ZoneId from '../../../../../resources/zone_id';
+import { OopsyData } from '../../../../../types/data';
+import { OopsyTriggerSet } from '../../../../../types/oopsy';
+
+export type Data = OopsyData;
+
+const triggerSet: OopsyTriggerSet<Data> = {
+  zoneId: ZoneId.MountOrdeals,
+  damageWarn: {
+    'Rubicante Circle of Purgatory Fiery Expiation': '7CC5', // conal from outer ring
+    'Rubicante Arch Inferno': '7CC8', // center puddle around boss
+    'Rubicante Inferno Devil': '7CCA', // 2x large rotating circle AoEs during Arch Inferno
+    'Rubicante Hellscathe': '7CCF', // small circle AoEs baited on players during Arch Inferno
+    'Rubicante Conflagration': '7CCB', // middle danger bacon during Arch Inferno
+    'Rubicante Radial Flagration': '7CCD', // large fan-style AoE during Arch Inferno
+    'Rubicante Flamesent Ghastly Wind': '7CD6', // targeted conal (adds phase)
+    'Rubicante Flamesent Ghastly Flame 1': '7CD7', // ground circle A0Es (adds phase)
+    'Rubicante Flamesent Ghastly Flame 2': '7CD8', // ground circle A0Es (adds phase)
+    'Rubicante Flamerake 1': '7CDF', // first of three line AoEs (center)
+    'Rubicante Flamerake 2': '7CE0', // second (pair) of three line AoEs (sides of boss)
+    'Rubicante Flamerake 3': '7CE1', // third (pair) of three line AoEs (edges)
+    'Rubicante Scalding Fleet': '7CE9', // baited proteans
+    'Rubicante Sweeping Immolation': '7CE3', // 180 front cleave
+  },
+  shareWarn: {
+    'Rubicante Explosive Pyre': '7CDB', // tank autos w/puddle cleave
```

Comment:
Is this an add phase ability? Or when does this happen? I've seen the flying text for this in extreme, but could not tell what it was.

Reply:
Post adds-phase, he becomes "empowered" with the blue wing (similar to P3 in Ex), and Explosive Pyre is his tank auto that does a small puddle cleave around the target.  (Somewhat annoyingly, he uses it only a second or two after total immolation resolves when the party is stacked up... 😆)

Reply:
Should this not be ignored in the timeline, then?

Reply:
He uses it pretty often, and it's not a significant amount of damage.  This is what would be added to Phase 2:

```
2050.6 "Explosive Pyre" sync / 1[56]:[^:]*:Rubicante:7CDB:/
2068.5 "Explosive Pyre" sync / 1[56]:[^:]*:Rubicante:7CDB:/
2071.6 "Explosive Pyre" sync / 1[56]:[^:]*:Rubicante:7CDB:/
2084.9 "Explosive Pyre" sync / 1[56]:[^:]*:Rubicante:7CDB:/
2087.9 "Explosive Pyre" sync / 1[56]:[^:]*:Rubicante:7CDB:/
2091.0 "Explosive Pyre" sync / 1[56]:[^:]*:Rubicante:7CDB:/
2102.2 "Explosive Pyre" sync / 1[56]:[^:]*:Rubicante:7CDB:/
2117.0 "Explosive Pyre" sync / 1[56]:[^:]*:Rubicante:7CDB:/
```

And this would get added to the loop:

```
2262.1 "Explosive Pyre" sync / 1[56]:[^:]*:Rubicante:7CDB:/
2265.2 "Explosive Pyre" sync / 1[56]:[^:]*:Rubicante:7CDB:/
2274.3 "Explosive Pyre" sync / 1[56]:[^:]*:Rubicante:7CDB:/
2289.6 "Explosive Pyre" sync / 1[56]:[^:]*:Rubicante:7CDB:/
2292.6 "Explosive Pyre" sync / 1[56]:[^:]*:Rubicante:7CDB:/
2295.7 "Explosive Pyre" sync / 1[56]:[^:]*:Rubicante:7CDB:/
2306.9 "Explosive Pyre" sync / 1[56]:[^:]*:Rubicante:7CDB:/
```

If you don't think that's too spammy, happy to add it in!

Reply:
haha oh gosh, nevermind